types/webextension-polyfill/namespaces/storage.d.ts

```diff
@@ -186,15 +186,15 @@ export namespace Storage {
     /**
      * Object mapping each key that changed to its corresponding $(ref:storage.StorageChange) for that item.
      */
-    interface StorageAreaOnChangedChangesType extends StorageChange {
-        [s: string]: unknown;
+    interface StorageAreaOnChangedChangesType<OType = any, NType = OType> {
+        [s: string]: StorageChange<OType, NType> | undefined;
```

Comment:
Dont think there should be undefined but i am not familiar with the APIs here so..
Is NType default to undefined instead ?